addons/zeus/functions/fnc_getModuleDestination.sqf

```diff
@@ -5,7 +5,14 @@
  * Arguments:
  * 0: The souce object <OBJECT>
  * 1: Code to run when position is ready <CODE>
- * - Code is passed [0: Successful <BOOL>, 1: Object <OBJECT>, 2: Position ASL <ARRAY>]
+ * - Code is passed [
+                        0: Successful <BOOL>, 
+                        1: Object <OBJECT>, 
+                        2: Position ASL <ARRAY>, 
+                        3: State of Shift <BOOL>, 
+                        4: State of Ctrl <BOOL>,
+                        5: State of Alt <BOOL>
```

Comment:
I don't like this indentation, I assume this is stuff passed on from higher scope? I think this is clearer either way:
```
 * - Code is passed <ARRAY>
 *   0: Successful <BOOL>
 *   1: Object <OBJECT>
 *   2: Position ASL <ARRAY>
 *   3: State of Shift <BOOL>
 *   4: State of Ctrl <BOOL>
 *   5: State of Alt <BOOL>
```
@SilentSpike How would you handle that in your doc script?